### PR TITLE
authoriser fixes add aws json serializer namespace

### PR DESCRIPTION
**Pull request by LBHBomotoso**
auth handler function specified in yml

### CODE REVIEW

File: ApiAuthTokenGenerator/LambdaHandler.cs [hunk 1]
```diff
@@ -25,6 +29,8 @@ public LambdaHandler(IServiceProvider serviceProvider)
 
         public APIGatewayCustomAuthorizerResponse VerifyToken(APIGatewayCustomAuthorizerRequest request)
         {
+            LambdaLogger.Log("token: " + request.AuthorizationToken + ":" + request.Headers["Authorization"]);
```
Comment (LBHMKeyworth) [inline ApiAuthTokenGenerator/LambdaHandler.cs:32]: I don't think we should log the tokens when this is in production - is this just for while you're getting it working?

Comment (LBHBomotoso) [reply ApiAuthTokenGenerator/LambdaHandler.cs:32]: I won't deploy this to production. Will keep in staging.

Comment (LBHBomotoso) [reply ApiAuthTokenGenerator/LambdaHandler.cs:32]: Also only using tokens generated in staging

Comment (elcorbs) [reply ApiAuthTokenGenerator/LambdaHandler.cs:32]: Maybe add a check here, to check the environment is staging just in case it does end up in prod

Comment (LBHBomotoso) [reply ApiAuthTokenGenerator/LambdaHandler.cs:32]: will do

**Pull request merged**